style.css

```diff
@@ -0,0 +1,55 @@
+main {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+}
+.item {
+  padding: 1rem;
+}
+.hero-content {
+  margin: 3rem auto;
+}
+.hero-image {
+  display: block;
+  max-width: 100%;
+  max-inline-size: 100%;
+  block-size: auto;
+}
+.hero-heading {
+  font-size: 4rem;
+  color: var(--primary-color);
+}
+.hero {
+  text-align: center;
+}
+.hero-text {
+  font-size: 1.5rem;
+}
+.btns-div {
+  margin-top: 3rem;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
```

Comment:
the space between buttons is much larger than expected I think we can give some width to the container to fix this